File: core/src/main/java/com/ustadmobile/core/view/ReportTopLEsDetailView.java
package com.ustadmobile.core.view;


/**
 * Core View. Screen is for ReportTopLEsDetail's View
 */
public interface ReportTopLEsDetailView extends ReportDetailView {


    // This defines the view name that is an argument value in the go() in impl.
    String VIEW_NAME = "ReportTopLEsDetail";

    //Any argument keys:

    /**
     * Method to finish the screen / view.
     */
    void finish();


}

File: core/src/commonMain/kotlin/com/ustadmobile/core/view/ReportTopLEsDetailView.kt
package com.ustadmobile.core.view


/**
 * Core View. Screen is for ReportTopLEsDetail's View
 */
interface ReportTopLEsDetailView : ReportDetailView {

    //Any argument keys:

    /**
     * Method to finish the screen / view.
     */
    override fun finish()

    companion object {


        // This defines the view name that is an argument value in the go() in impl.
        val VIEW_NAME = "ReportTopLEsDetail"
    }


}

